x-pack/legacy/plugins/uptime/public/pages/page_header.tsx

```diff
@@ -60,6 +66,11 @@ export const PageHeader = ({ headingText, breadcrumbs, datePicker = true }: Page
             <h1>{headingText}</h1>
           </EuiTitle>
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <Link to={`${SETTINGS_ROUTE}`}>
+            <EuiButton data-test-subj="settings-page-link">{settingsLinkText}</EuiButton>
```

Comment:
```suggestion
            <EuiButtonEmpty data-test-subj="settings-page-link">{settingsLinkText}</EuiButtonEmpty>
```

Reply:
The settings button also appears in the header of the settings view. I would remove it there, it is not needed.

Reply:
Eventually that area will contain alerts and ML. Am I right that those buttons should only be present on the overview? I think I'll name the option something like "extra_buttons" and only enable it on the overview. Does that make sense.